### PR TITLE
refactor: Simplify experimental Schelling example

### PR DESCRIPTION
**Pull request by rht**

### CODE REVIEW

**Comment by rht** [comment]
@projectmesa/maintainers this PR needs to be merged so that the Docker example in the core repo works again.

**Review by Corvince — APPROVED** [review]

**Comment by Corvince** [comment]
Thank you! I don't know if you want to squash and commit this or not, so I leave this up to you

**Pull request merged**
